server.lua

```diff
@@ -399,6 +401,76 @@ function GetGuildRoleList(guild --[[optional]])
 end
 
 recent_role_cache = {}
+function FindGuildForRole(roleId)
+	local roleFound
+	local tempGuildList = Config.Guilds
+	tempGuildList["main_guild_abcd1010"] = Config.Guild_ID

```

Comment:
Lol, what is the need for this arbitrary key in the table?

Reply:
Mostly to make it unlikely that a user would overwrite that key in the config by 'injecting the main guild ID in a nonsensical key